src/clickableText.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Support for clickable text in instructions.
+ *
+ * When the text markdown contains:
+ *   [text](#clickable=id)
+ *
+ * This code will generate
+ *   <b data-id="id" class="clickable-text">text</b>
+ */
+
+module.exports = function clickableText() {
+  const Parser = this.Parser;
+  const tokenizers = Parser.prototype.inlineTokenizers;
+  const originalText = tokenizers.link;
+  tokenizers.link = function (eat, value, silent) {
+    const link = originalText.call(this, eat, value, silent);
+    if (link && link.url && link.url.startsWith("#clickable=")) {
+      const regexp = /#clickable=(.*)/;
```

Comment:
nit; I think we'll get a slight performance improvement if we instantiate this regex outside the scope of these functions. See for example https://github.com/code-dot-org/remark-plugins/blob/d8d480504fdf36350fa8b802b7883c3a2b7b0718/src/blockfield.js#L3

Reply:
Good call.  Done.